datadriven/src/sgpp/datadriven/datamining/configuration/DataMiningConfigParser.cpp

```diff
@@ -34,6 +34,7 @@
 #include <sgpp/datadriven/datamining/modules/fitting/FitterTypeParser.hpp>
 #include <sgpp/datadriven/datamining/modules/scoring/ScorerMetricTypeParser.hpp>
 #include <sgpp/solver/TypesSolver.hpp>
+#include <sgpp/datadriven/datamining/modules/visualization/VisualizationTypesParser.hpp>
 
 #include <map>
```

Comment:
Add missing `#include`.
```suggestion
#include <iostream>
#include <map>
```